utils/prometheusmetrics/prometheusmetrics.go

```diff
@@ -0,0 +1,229 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//      http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package prometheusmetrics
+
+import (
+	"net/http"
+	"strconv"
+	"sync"
+	"time"
+
+	"github.com/aws/amazon-vpc-cni-k8s/pkg/utils/logger"
+	"github.com/aws/amazon-vpc-cni-k8s/pkg/utils/retry"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+)
+
+var log = logger.Get()
+
+var (
+	IpamdErr = prometheus.NewCounterVec(
```

Comment:
I like that these are centralized now. I'm wondering if we can turn these descriptions into a doc/README so that anyone getting metrics from `aws-node` or `cni-metrics-helper` can reference a doc to see everything they have available

Reply:
Yeah we need to have since generally we get those questions but let's do as a fast follow up